### PR TITLE
Improve recycling day loop

### PR DESCRIPTION
**Pull request by GeekOnCoffee**
* Rename `failsafe_count` to `days_counted` to better indicate what's happening
* Move recycling_date strip out of loop

### CODE REVIEW

File: appletonapi.py [hunk 1]
```diff
@@ -220,22 +220,23 @@ def execute(self, address, user_agent):
             garbage_day = prop_response['result'][1]['garbageday']
             recycling_day = prop_response['result'][1]['recycleday']
             split_recycling_day = recycling_day.split(',')
+            recycling_date = split_recycling_day[1].strip()
```
Comment (mikeputnam) [inline appletonapi.py]: Good idea, thanks.

File: appletonapi.py [hunk 2]
```diff
@@ -220,22 +220,23 @@ def execute(self, address, user_agent):
...
             found_recycling = False
             cur_date = datetime.now()
-            failsafe_count = 0
+            days_counted = 0
```
Comment (mikeputnam) [inline appletonapi.py]: Hey @mrosack what do you think about this?

The way I interpret this while loop, it is looking forward 21 days to fetch future pickup days. It is also a failsafe so the loop doesn't go too far/too short/infinitely.

What about a variable like `lookahead_days`?

Comment (mrosack) [inline appletonapi.py]: Agreed - a variable would make the intention more clear there, good idea!

Comment (mikeputnam) [inline appletonapi.py]: @GeekOnCoffee Could you refactor this pull request to use `lookahead_days` instead of `days_counted`?

Otherwise :+1:

Comment (GeekOnCoffee) [inline appletonapi.py]: @mikeputnam all set.

**Comment by mikeputnam** [comment]
Thanks for the code! I'll get this deployed.

**Pull request merged**